### ISSUE TITLE
@partial-block is overridden

### ISSUE THREAD

**Issue by btmorex** [wycats/handlebars.js#1099]
Given test.hbs:

```
{{#> layout}}Outer{{/layout}}
```

and layout.hbs:

```
{{#> inner}}Inner{{/inner}}
{{> @partial-block }}
```

I would expect:

Inner
Outer

but I get:

Inner
Inner

**Comment by kpdecker** [comment]
I think I know what is going on here. Should be a relatively simple fix. Will take a look!

**Comment by kpdecker** [comment]
Released in 4.0.3
